content/documentation/pages/8-resources/1-reference-docs.md

```diff
@@ -15,10 +15,9 @@ The following projects play a pivotal part in the streaming and batch data proce
 The projects all individually evolve with different backlogs, priorities, and release cadences. However, Spring Cloud Data Flow as a product brings them together for a cohesive developer experience.
 Hence, we broadly refer to these projects as the "Spring Cloud Data Flow Ecosystem".
 
-| Name                                                                                          | Description                                                                                         | Documentation                                                                                                               |
```

Comment:
This is really part of https://github.com/spring-io/dataflow.spring.io/issues/406